code/ship/shipfx.h

```diff
@@ -122,25 +122,24 @@ void shipfx_engine_wash_level_init();
 // pause engine wash sounds
 void shipfx_stop_engine_wash_sound();
 
-#define WD_NONE		0
-#define WD_WARP_IN	1
-#define WD_WARP_OUT	2
-float shipfx_calculate_warp_time(object *objp, ship_info *sip, int warp_dir, float half_length, float warping_dist);
+enum class WarpDirection { WD_NONE, WD_WARP_IN, WD_WARP_OUT };
```

Comment:
Since the enum class has its own scope the WD_ prefix isn't needed anymore and could be removed to make this easier to read.

Reply:
Mmm, true.